Source/Relativity.Test.Helpers/Relativity.Test.Helpers/packages.config

```diff
@@ -1,7 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="BCrypt-Official" version="0.1.109" targetFramework="net462" />
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net462" />
+  <package id="BouncyCastle" version="1.8.5" targetFramework="net462" />
+  <package id="MailKit" version="2.2.0" targetFramework="net462" />
+  <package id="MimeKit" version="2.2.0" targetFramework="net462" />
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net462" />

```

Comment:
If these nuget packages are need for TestHelpers to work, we need to make them as a dependency for TestHelpers nuget package.

Reply:
I have these noted, we will need to coordinate on this afterwards